docs/DeveloperGuide.md

```diff
@@ -341,6 +341,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | user with many contacts in the address book   | search for contacts by their name                                                            | locate details of persons without having to go through the entire list                       |                                              |
 | `* *`    | user                                          | add a picture to my contacts                                                                 | easily identify my contacts and add a personal touch to them                                 |
 | `* *`    | user                                          | hide private contact details                                                                 | minimize chance of someone else seeing them by accident                                      |
+| `* *`    | user                                          | quickly view the available commands                                                          | view quick command help without needing to leave the application                             |
```

Comment:
is this supposed to be 

'quickly view the available commands without needing to leave the application'?

Reply:
> As a user, I want to quickly view the available commands, so that I can quickly view the available commands without needing to leave the application.

vs

> As a user, I want to quickly view the available commands, so that I can view quick command help without needing to leave the application.

I deliberately chose to change the phrasing of the 3rd part of the user story in an attempt to make it sound less repetitive.

Reply:
ah i see, the phrase 'quick command help' threw me off, otherwise LGTM